structuring-content/problem-set-structuring-content.md

```diff
@@ -389,11 +389,21 @@ Do this work in a new `index.html` file on your computer, then reflect and answe
 
 Do not spend more than three hours on this. At three hours, consider it done! There is no submission for this. Be prepared to share your thought process with others.
 
+### !callout-info
+
+## Editing and Viewing HTML Files
+
+VS Code is a great editor for HTML files! It provides syntax hilighting and tag matching which makes writing your markup easier. 
+
+When you want to see how your HTML file is rendered, paste the full path to your file in the navigation bar of your browser. You can get the full path to a file in VS Code by holding control and clicking the name of the file, then selecting 'Copy Path'.
```

Comment:
You may need to explain how to get the full path to a file here.  Most students are still iffy on filesystem paths in Mac/Unix systems.  

TBF most don't get it in Windows either.

Reply:
I added the note about how to get the path in VS Code. Do you think I should also add instructions about how to do it using `pwd` instead?

Reply:
Knowing `pwd` is useful, but not essential.

Reply:
I agree. I think `pwd` might be a bit much to cram in here. So there's no change needed, right? The explanation of how to get the path from VS Code should be sufficient?

Reply:
```suggestion
When you want to see how your HTML file is rendered, paste the full path to your file in the navigation bar of your browser. You can get the full path to a file in VS Code by holding `control` and clicking the name of the file (or right-clicking the file with a mouse), then selecting 'Copy Path'.
```

what do you think?